gravity_upgrade.md

```diff
@@ -72,25 +73,126 @@ A manual upgrade operation starts with an operation plan which is a tree of acti
+      run the actual containers used by Kubernetes to run its own Pods.
+    - `sudo journalctl -f -u 'registry'`
+      This command will show all logs from the container registry, which is used to
+      distribute containers' data among all nodes.
+    - `sudo journalctl -f -u 'planet-agent'`
+      This command will show all logs from the planet-agent daemon, which is used to
+      exchange messages in between the different nodes in Gravity clusters.
+
+### Collect logfiles for reporting purpose
+  + `sudo gravity report reportfile.tgz`
+  + content of `gravity-system.log` and (if present) `gravity-install.log` files.
+    These files are usually found in the installation directory
+    or in newer versions they will be located in /var/log/gravity
+
+### Nifty features coming in future versions
+Versions of Gravity from 6.1.x onward also includes a nice feature showing the
+status changes over time which may be helpful to identify root causes or brief
+fluctuations of the cluster state:
 
+```
+ubuntu@telekube0:~/i$ sudo gravity status history
+2020-07-30T16:18:08Z [Leader Elected]   new leader 172.28.128.3
+2020-07-30T16:18:35Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:19:13Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=dns
+2020-07-30T16:19:13Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:19:13Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:19:35Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:19:35Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=dns
+2020-07-30T16:19:35Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:29:38Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:29:38Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:30:06Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:30:06Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:30:57Z [Node Degraded]    node=172_28_128_3.pensivevillani4966
+2020-07-30T16:31:07Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:32:05Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:32:05Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+2020-07-30T16:32:46Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:32:46Z [Probe Failed]     node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:33:07Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=node-status
+2020-07-30T16:34:05Z [Probe Succeeded]  node=172_28_128_3.pensivevillani4966    checker=etcd-healthz
+2020-07-30T16:34:05Z [Node Healthy]     node=172_28_128_3.pensivevillani4966
+```
 
```

Comment:
I would add a few other things from `5.5.49` and `5.5.50` - that pertain to customer `M`

- **5.5.49:** `gravity status` - now display unhealhty critical system pods ([#1707](https://github.com/gravitational/gravity/pull/1707)) + added visibility into implicit controller status loop if cluster is in degraded state ([#1707](https://github.com/gravitational/gravity/pull/1707))
- **5.5.49:** Should we mention anything about pre-upgrade checks? If so [#1731](https://github.com/gravitational/gravity/pull/1731) added check to see if previous upgrade operation has succeeded/if failed plan was rolled back.
- **5.5.50:** mention `gravity plan resume` launching in background job to execute phases by default, adding `--block` resume plan in foreground, and the new command `gravity plan watch` to track the plan progress ([#1807](https://github.com/gravitational/gravity/issues/1807))
- **5.5.50:** ability to view/collect gravity cli command history `journalctl -t gravity-cli` ([#1799](https://github.com/gravitational/gravity/issues/1799))
- **5.5.50:** pre-upgrade check to see if teleport nodes are unavailable ([#1817](https://github.com/gravitational/gravity/issues/1817))

Reply:
I think most of this have now been added to the new commit. Closing this for now.